tests/test_permutations.py

```diff
@@ -0,0 +1,20 @@
+from delphi.train.permute import Permutation
+
+
+def test_permute():
+
+    perm = Permutation(1)
+
+    # Test the random number generator
+    for i in range(100000):
+        assert perm.random_number_generator() >= 0
+        assert perm.random_number_generator() <= 1
+    state = perm.state
+    assert state == 822144001
+    # Test the permutation
+    test_list = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+    length = len(test_list)
+    for i in range(100000):
+        perm.permute(test_list)
+        assert len(test_list) == length
```

Comment:
you could check if it's a permutation, i.e. all elements are present before and after